src/platform/P6/BLEManagerImpl.cpp

```diff
@@ -787,7 +789,7 @@ void BLEManagerImpl::SetAdvertisingData(void)
     ChipBLEDeviceIdentificationInfo mDeviceIdInfo;
     uint16_t deviceDiscriminator = 0;
     uint8_t localDeviceNameLen;
-    uint8_t service_data[9];
+    uint8_t service_data[BLE_SERVICE_DATA_SIZE];
```

Comment:
Would using `sizeof(ChipBLEDeviceIdentificationInfo) + 2` be less-error prone? Or at least a static_assert asserting that equality?

Reply:
Thanks for the review.. I added static_assert check for the same.